design/resources.go

```diff
@@ -324,6 +330,7 @@ var _ = Resource("trackerquery", func() {
 	})
 
 	Action("create", func() {
+		Security("jwt")
```

Comment:
May I ask if tests for trackerQuery are to be added later ? is the work in progress ?